README.md

```diff
@@ -749,10 +749,40 @@ Please see the [`getrlimit` linux manpage](https://man7.org/linux/man-pages/man7
 
 The full list of resource limits that can be configured in Bottlerocket are:
 
-resource limit | setting | default value
------ | ----- | -----
-`RLIMIT_NOFILE` | `settings.oci-defaults.resource-limits.max-open-files.hard-limit` | 1048576
-`RLIMIT_NOFILE` | `settings.oci-defaults.resource-limits.max-open-files.soft-limit` | 65536
+resource limit | setting | default value | unit
+----- | ----- | ----- | -----
+`RLIMIT_AS` | `settings.oci-defaults.resource-limits.max-address-space.hard-limit` | - | bytes
+`RLIMIT_AS` | `settings.oci-defaults.resource-limits.max-address-space.soft-limit` | - | bytes
```

Comment:
If we use an HTML table instead, we could use `rowspan=2` for the resource limit and unit fields to avoid duplicating those values. Might be worth trying out to see if the result seems more polished.